packages/snap/src/index.ts

```diff
@@ -172,11 +245,21 @@ export const processRpcRequest: SnapRpcProcessor = async (
         holder.encryptionPrivKey,
       );
 
+      let zkCertificateName: string; // Default name
+      if (zkCert.zkCertStandard === ZkCertStandard.TwitterZkCertificate) {
+        zkCertificateName = 'Twitter ZK Certificate';
+      } else if (zkCert.zkCertStandard === ZkCertStandard.ZkKYC) {
+        zkCertificateName = 'ZkKYC';
+      } else {
+        throw new Error(`Unsupported zkCertificate type`);
```

Comment:
In my opinion the snap needs to support future unknown zkCert types because external devs can not modify the snap.
Instead of getting setting a name here, you can just use the value of `zkCert.zkCertStandard` in the text below.